src/pointFeature.js

```diff
@@ -2,13 +2,59 @@ var inherit = require('./inherit');
 var feature = require('./feature');
 
 /**
- * Create a new instance of class pointFeature
+ * Object specification for a point feature.
+ *
+ * @typedef {geo.feature.spec} geo.pointFeature.spec
+ * @property {geo.geoPosition|function} [position] Position of the data.
+ *   Default is (data).
+ * @property {object} [style] Style object with default style options.
+ * @property {number|function} [style.radius=5] Radius of each point in pixels.
+ *   This is the fill radius inside of the stroke.
+ * @property {boolean|function} [style.stroke=true] True to stroke point.
+ * @property {geo.geoColor|function} [style.strokeColor] Color to stroke each
+ *   point.
+ * @property {number|function} [style.strokeOpacity=1] Opacity for each point's
+ *   stroke.  Opacity is on a [0-1] scale.
+ * @property {number|function} [style.strokeWidth=1.25] The weight of the
+ *   point's stroke in pixels.
+ * @property {boolean|function} [style.fill=true] True to fill point.
+ * @property {geo.geoColor|function} [style.fillColor] Color to fill each
+ *   point/
+ * @property {number|function} [style.fillOpacity=1] Opacity for each point.
+ *   Opacity is on a [0-1] scale.
+ * @property {boolean|geo.pointFeature.clusteringSpec} [clustering=false]
+ *   Enable point clustering.
+ * @property {string} [primitiveShape='sprite'] For the gl renderer, select the
+ *   primitive shape.  This is one of `'triangle'`, `'square'`, or `'sprite'`.
+ *   `sprite` uses the least memory, `triangle` is fastest if the vertex shader
+ *   is the bottleneck, and `square` is fastest if the fragment shader is the
+ *   bottleneck.  `sprite` may not work for very large points.
+ * @property {boolean} [dynamicDraw=false] For the gl renderer, if this is
+ *   truthy, webgl source buffers can be modifies and updated directly.
+ */
+
+/**
+ * Point clustering specification.
+ *
+ * @typedef {object} geo.pointFeature.clusteringSpec
+ * @property {number} [radius=0.05] This is combined with the `width` and
+ *   `height` to determine how close points need to be to each other to be
+ *   clustered.
+ * @property {number} [maxZoom=18] Never cluster above this zoom level.
+ * @property {number} [width=256]
+ * @property {number} [height=256]
+ */
+// TODO: refactor point clustering to (a) not be based on window size, (b) use
```

Comment:
perhaps we could create an issue for this and remove it from here?

Reply:
Sure.

Reply:
Apparently I already did (see issue #853).  Do you want it removed?  It stands as a warning to anyone using point clustering.

Reply:
I would remove it as it is not consistent and a user would not dig in the code as much I think.

Reply:
Done.